admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -0,0 +1,425 @@
+   package repository.
+
+   .. note:: Skip this step if you are planning to run Archivematica in indexless
+      mode (without Elasticsearch).
+
+   .. code:: bash
+
+      sudo wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | sudo apt-key add -
+      sudo sh -c 'echo "deb http://packages.elasticsearch.org/elasticsearch/1.7/debian stable main" >> /etc/apt/sources.list'
+
+   At this point you will need to restart dashboard and mcp-server services
+
+   .. code:: bash
+
+      sudo service archivematica-dashboard restart
+      sudo service archivematica-mcp-server restart
+
+
+3. Update to the most recent OS release - for Xenial, this is 16.04.4. This step
+   will also fetch a list of the software from the package repositories you just
+   added to your system.
+
+   .. code:: bash
+
+      sudo apt-get update
+      sudo apt-get upgrade
+
+4. Install Elasticsearch (optional)
+
+   .. note:: Skip this step if you are planning to run Archivematica in indexless
+      mode (without Elasticsearch).
+
+   .. code:: bash
+
+      sudo apt-get install elasticsearch
+
+5. Install the Storage Service package.
+
+   .. code:: bash
+
+      sudo apt-get install -y archivematica-storage-service
+
+6. Configure the Storage Service.
+
+   .. code:: bash
+
+      sudo rm -f /etc/nginx/sites-enabled/default
+      sudo ln -s /etc/nginx/sites-available/storage /etc/nginx/sites-enabled/storage
+
+   .. warning:: If you are planning to use the `Sword API`_ of the Archivematica
+      Storage Service, then (due to a `known issue`_), you must instruct
+      Gunicorn to use the ``sync`` worker class:
+
+   .. code:: bash
+
+      sudo sh -c 'echo "SS_GUNICORN_WORKER_CLASS=sync" >> /etc/default/archivematica-storage-service'
+
+7. Update pip. This is used to install Python dependencies for both the Storage
+   Service and the dashboard. There is a `known issue with pip`_ on Ubuntu 14.04
+   that makes this step necessary. This step is optional on Ubuntu 16.04, but is
+   still a good idea to get the most recent version of pip.
+
+   .. code:: bash
+
+      sudo wget https://bootstrap.pypa.io/get-pip.py
+      sudo python get-pip.py
+
+8. Install the Archivematica packages. The order of installation is important -
+   the mcp-server package must be installed before the dashboard package. While
+   it is possible to install the mcp-client package on a separate machine, that
+   configuration is not documented in these instructions.
+
+   The mcp-server package will install MySQL and configure the database used by
+   Archivematica. Depending on the version of MySQL that gets installed the
+   prompts you will see may differ. In all cases, you will be prompted to create
+   a password for the 'root' user. Keep note of the password you create.
+   On Ubuntu 14.04, MySQL 5.5 is installed, and
+   the default 'archivematica' database user is automatically created with a
```

Comment:
Inline literals in RST use double backtick, see http://docutils.sourceforge.net/docs/user/rst/quickref.html.